pocket_coffea/executors/executors_lxplus.py

```diff
@@ -287,24 +257,55 @@ def recreate_jobs(self, jobs_to_recreate):
         with open(f"{self.jobs_dir}/jobs_config.yaml") as f:
             jobs_config = yaml.safe_load(f)
 
+        if jobs_to_recreate == "auto":
+            jobs_to_recreate = [f.replace(".failed","") for f in os.listdir(self.jobs_dir) if f.endswith(".failed")]
```

Comment:
This is very good!! I implemented the resubmission only in the `check_jobs` script, https://github.com/PocketCoffea/PocketCoffea/blob/main/pocket_coffea/scripts/check_jobs.py#L99,  but having it in the recreation of the jobs can be very useful! 

In principle is not needed to recreate jobs thought to just resubmit them. You can use the check_jobs script.